### ISSUE TITLE
Update 1.0.0.1 halts

### ISSUE THREAD

**Issue by OldFcuk** [BlueByteSystemsInc/PDMCLI-QA#3]
Update stays at this window
![image](https://user-images.githubusercontent.com/84081713/169599248-ff2298cf-0c81-4ef1-876f-1dd138445b35.png)
It continues after closing this.

Installer says repair instead of update
![image](https://user-images.githubusercontent.com/84081713/169599522-27b6654b-4ec1-421c-9d86-56a07c7f5059.png)

No start menu is item created.

After upgrade/repair it still states about new version
![image](https://user-images.githubusercontent.com/84081713/169600105-d124e58c-9d20-4a3e-a73c-33c2ff50b95c.png)

**Comment by AmenJlili** [comment]
@OldFcuk : Yup. I think the version on the server is actually 1.0.0 - Will fix that.

**Comment by AmenJlili** [comment]
Fixed.

**Comment by AmenJlili** [comment]
This issue is not resolved yet.
